shared/login/join-or-login/index.tsx

```diff
@@ -1,82 +1,77 @@
 import * as React from 'react'
 import * as Kb from '../../common-adapters'
 import * as Styles from '../../styles'
+import {InfoIcon} from '../../signup/common'
 
 type Props = Kb.PropsWithTimer<{
   bannerMessage: string | null
   checkIsOnline: () => void
-  onFeedback: (() => void) | null
   onLogin: () => void
   onSignup: () => void
   isOnline: boolean | null
   showProxySettings: () => void
 }>
 
-const Feedback = ({onFeedback}) =>
-  onFeedback ? (
-    <Kb.Text type="BodySmallSecondaryLink" onClick={onFeedback}>
-      Problems logging in?
-    </Kb.Text>
-  ) : (
-    <Kb.Text type="BodySmall">
-      Send us feedback! Run{' '}
-      <Kb.Text type="TerminalInline" selectable={true}>
-        keybase log send
-      </Kb.Text>{' '}
-      from the terminal.
-    </Kb.Text>
-  )
-
 const Intro = (props: Props) => {
   Kb.useInterval(() => props.checkIsOnline, 2000)
```

Comment:
Oh, this is wrong. `() => props.checkIsOnline()`

Reply:
We should add a `useEffect` call to this too, otherwise "Configure a proxy" shows for 2s until we check the first time.

Reply:
Found a better way to handle it, during the first 2s the prop is undefined rather than false.